### PR TITLE
Set the default count for section template_tags to None

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@saeedmb8 @miltontony  ready for review

**Comment by saeedmb8** [comment]
👍  form me once travis is happy

File: molo/core/templatetags/core_tags.py [hunk 1]
```diff
@@ -170,19 +170,21 @@ def load_descendant_articles_for_section(
...
     Returns all child articles
     If the `locale_code` in the context is not the main language, it will
     return the translations of the live articles.
     '''
     locale = context.get('locale_code')
     p = context.get('p', 1)
-
     qs = section.articles()
 
+    if not count:
```
Comment (miltontony) [inline molo/core/templatetags/core_tags.py]: This is the same as setting `count=1` in the method definition

File: molo/core/tests/test_models.py [hunk 2]
```diff
@@ -104,6 +104,12 @@ def test_image(self):
         self.assertEquals(
             new_section3.get_effective_image(), new_section.image)
 
+    def test_number_of_child_sections(self):
+        new_section = self.mk_section(self.section_index)
+        self.mk_sections(new_section, count=12)
+        response = self.client.get('/sections/test-section-0/')
+        self.assertContains(response, 'Test Section 11')
```
Comment (miltontony) [inline molo/core/tests/test_models.py]: You probably need to test a few more things - you're currently only testing that Section 11 is showing - you need to test that *only* Section 11 is show. You can do that by asserting that other sections are not there.

File: molo/core/templatetags/core_tags.py (hunk 1 above)
Comment (codiebeulaine) [inline molo/core/templatetags/core_tags.py]: changing now :)

File: molo/core/tests/test_models.py (hunk 2 above)
Comment (codiebeulaine) [inline molo/core/tests/test_models.py]: @miltontony I want the other sections to be there, I want to test that it doesn't stop showing sections after the fifth secion

File: molo/core/templatetags/core_tags.py [hunk 3]
```diff
@@ -170,19 +170,19 @@ def load_descendant_articles_for_section(
...
     If the `locale_code` in the context is not the main language, it will
     return the translations of the live articles.
     '''
     locale = context.get('locale_code')
     p = context.get('p', 1)
-
     qs = section.articles()
 
     # Pagination
     paginator = Paginator(qs, count)
```
Comment (miltontony) [inline molo/core/templatetags/core_tags.py]: @codiebeulaine doesn't passing 1 in here result in just 1 section per page?

File: molo/core/templatetags/core_tags.py [hunk 4]
```diff
@@ -142,7 +142,7 @@ def render_translations(context, page):
 @register.assignment_tag(takes_context=True)
 def load_descendant_articles_for_section(
         context, section, featured_in_homepage=None, featured_in_section=None,
-        featured_in_latest=None, count=5):
+        featured_in_latest=None, count=None):
```
Comment (miltontony) [inline molo/core/templatetags/core_tags.py]: This should remain `5`. Since there's pagination for articles.

**Comment by miltontony** [comment]
👍 once travis 🍰

**Pull request merged**